idconn/__init__.py

```diff
@@ -10,7 +10,6 @@
 
 with warnings.catch_warnings(record=True) as w:
     warnings.simplefilter("ignore")
-    from . import idconn
```

Comment:
This reads as a circular import. Could be due to `idconn.py` being renamed to `pipeline.py` _or_ it could happen regardless. I haven't tested it out.